### ISSUE TITLE
custom format size doesn't save 'decimals behind the dot'' 

### ISSUE THREAD

**Issue by michpas** [Radarr/Radarr#6147]
**Describe the bug**
Radarr v3 doesn't save the broken number (eg. 2,5) as minimal or maximal filesize. It rounds it all off.

!-- A clear and concise description of what the bug is. -->
In Radarr v2, when creating customformat filters on filesize, broken numbers for minimal and maximum filesize (eg. 2,5) are accepted and saved.
When upgrading to Radarr v3, this config is correctly passed to v3.
However when a new value in V3 is entered it only saves the full rounded number.

**To Reproduce**
Goto customfilters
Add size as filter
Enter 4,5 (or I get 4.5 dependent on system default)
Press save
Reopen screen and value will be 4.

**Expected behavior**
To save the full entered value (4,5) (as it was in v2) and not round to 4 or to allow mb value instead of gb.

**Screenshots**
Added 2 images. Both of radar v3.
- One with imported profile from v2 (with value behind dot)
- and one newly in v3 created (where its impossible to indicate a broken number)

**Platform Information (please complete the following information):**
 - OS: Synology DS218+ 
 - Using latest Docker master version: 3.0.2.4552-ls94 by linuxserver.io
 
![113541225-1442c600-95e2-11eb-9221-47020ef5543f](https://user-images.githubusercontent.com/66222810/113621094-81467200-965b-11eb-89fe-716044d10a3b.jpg)
![113541227-14db5c80-95e2-11eb-96ee-bfa08eb73add](https://user-images.githubusercontent.com/66222810/113621147-93c0ab80-965b-11eb-85d7-d582cab524b5.jpg)

**Comment by ItsME6969** [comment]
Yes, not sure if it's a bug or was intentional but you can only make changes in 1GB increments which is a huge gap. Maybe drop it down to .1GB?

**Comment by bakerboy448** [comment]
Sonarr pull https://github.com/Radarr/Radarr/issues/9043
